src/python/espressomd/accumulators.py

```diff
@@ -253,24 +253,41 @@ class Correlator(ScriptInterfaceHelper):
     _so_name = "Accumulators::Correlator"
     _so_bind_methods = (
         "update",
+        "shape",
         "finalize")
     _so_creation_policy = "LOCAL"
 
     def result(self):
         """
+        Get correlation.
+
         Returns
         -------
+        :obj:`ndarray` of :obj:`float`
+            The result of the correlation function. The shape of the array
+            is determined by the shape of the input observable(s) and the
+            correlation operation.
+        """
+        return np.array(self.call_method(
+            "get_correlation")).reshape(self.shape())
 
-        numpy.ndarray
-            The result of the correlation function as a 2d-array.
-            The first column contains the values of the lag time tau.
-            The second column contains the number of values used to
-            perform the averaging of the correlation. Further columns contain
-            the values of the correlation function. The number of these columns
-            is the dimension of the output of the correlation operation.
+    def correlation_lags(self):
```

Comment:
I think the ```correlation_``` part is redundant because it's a member of ```Correlator```

Reply:
good point